pkg/cli/service.go

```diff
@@ -844,17 +844,21 @@ func (cve cveResult) stringYAML() (string, error) {
 type fixedTags struct {
 	Errors []errorGraphQL `json:"errors"`
 	Data   struct {
-		ImageList []imageStruct `json:"ImageListWithCVEFixed"` //nolint:tagliatelle // graphQL schema
+		PaginatedImagesResult PaginatedImagesResult `json:"ImageListWithCVEFixed"` //nolint:tagliatelle // graphQL schema
 	} `json:"data"`
 }
 
 type imagesForCve struct {
 	Errors []errorGraphQL `json:"errors"`
 	Data   struct {
-		ImageList []imageStruct `json:"ImageListForCVE"` //nolint:tagliatelle // graphQL schema
+		PaginatedImagesResult PaginatedImagesResult `json:"ImageListForCVE"` //nolint:tagliatelle // graphQL schema
 	} `json:"data"`
 }
 
+type PaginatedImagesResult struct {
```

Comment:
Is it possible to update imageListStructForDerivedImagesGQL/imageListStructForBaseImagesGQL at line 894 and 901 to use this struct instead of DerivedImageList  and BaseImageList